app/controllers/admin/conferences_controller.rb

```diff
@@ -82,7 +82,7 @@ def update
       short_title = @conference.short_title
       @conference.assign_attributes(conference_params)
       send_mail_on_conf_update = @conference.notify_on_dates_changed?
-      delete_event_schedules if @conference.start_hour_changed? || @conference.end_hour_changed?
+      delete_event_schedules if @conference.valid? && (@conference.start_hour_changed? || @conference.end_hour_changed?)
```

Comment:
what about moving it to a `after_update` callback? You should be able to use `changed?` there and it is much more clear.